nav2_mppi_controller/src/critics/goal_critic.cpp

```diff
@@ -21,6 +21,9 @@ namespace mppi::critics
 
 void GoalCritic::initialize()
 {
+  auto getParentParam = parameters_handler_->getParamGetter(parent_name_);
+  getParentParam(enforce_path_inversion_, "enforce_path_inversion", false);
```

Comment:
I think this should be using the `name_` getter instead. This is a goal critic plugin specific parameterization